portfolio/src/main/webapp/script.js

```diff
@@ -19,10 +19,15 @@ function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+  const visit = 
+      ['Tokyo', 'London', 'Rome', 'New Amsterdam', 'Taipei', 'Sichuan'];
+
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
+  const fact = visit[Math.floor(Math.random() * visit.length)];
```

Comment:
This should also be more descriptive. It doesn't seem like it will ever be a fact about you. It is a place you would like to visit, which with the context of "I would like to visit this place" become a fact. You might do what the original code did and use the singular case of the array name from which you are selecting.